src/openrct2/localisation/StringIds.h

```diff
@@ -3919,7 +3919,8 @@ enum
     STR_TILE_INSPECTOR_WALL_ANIMATION_FRAME = 6367,
 
     // Have to include resource strings (from scenarios and objects) for the time being now that language is partially working
-    STR_COUNT = 32768
+    // MAX_STR_COUNT - upper limit for number of strings, not the current count strings
+    MAX_STR_COUNT = 32768
```

Comment:
Why do we even have this value? Renaming it breaks nothing, why not just change it to
```suggestion
    MAX_STR_COUNT_DO_NOT_CHANGE = 32768
```